pvc_to_pvs_image_import/_topics/preparing_powervc_node.md

```diff
@@ -0,0 +1,222 @@
+### General Assumptions:
+
+1. You have the latest installation of the {{ site.data.product.title_short }} base (>= [Morphy](https://github.com/{{ site.data.product.title_short }}/manageiq/milestone/4)) on your server or workstation.
+
+
+
+
+2. You have `ansible-runner` installed in the system that hosts your {{ site.data.product.title_short }} instance.  
+For installation instructions please refer to this [guide](https://ansible-runner.readthedocs.io/en/stable/install.html). 
+
+
+3. Your {{ site.data.product.title_short }} base is located under `<MIQ_PATH>` on your {{ site.data.product.title_short }} hosting site.
+ 
+
+4. You have chosen a PowerVC control server for usage with the import workflow and have administrative access via it's command line.
+
+
+
+### Preparing IBM PowerVC Server (required once):
+All commands in this paragraph are to be run on a PowerVC server that is dedicated to supporting the image import workflow.
+This server should have `powervc-image` client application available in it's command line, typically included in the PowerVC installation. 
+
+1. Install `python3.X`, `pip3`, `virtualenv` applications as well as the `libselinux-python3` library.
+The standard `python3.X` installation will be used in order to run auxiliary scripts, which will 
+upload PowerVC ([OVA](https://en.wikipedia.org/wiki/Open_Virtualization_Format)) disk images to the [IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage). 
+The `pip3` is required in order to install Cloud Object Storage [Python SDK](https://github.com/ibm/ibm-cos-sdk-python) and auxiliary packages for proper functioning of the import workflow.
+The role of `virtualenv` will be to isolate python package installations that are relevant only to the image import workflow from the
+Python system packages. The `libselinux-python3` library is needed for the proper functioning of Ansible's `copy` module, which will
+be used for the copying of auxiliary data between the {{ site.data.product.title_short }} hosting server and the PowerVC server.
+   
+
+
+2. Prepare enough local storage for holding transient PowerVC disk images that will be transported into PowerVS
```

Comment:
Can we make this a function of the image size in a single sentence?  For example... "If you're attempting to import an image whose size is 20 GB, then you need 20 GB *2 + 100 MB (or whatever... I just made that up).  The other detail is good, but want to make it very clear though as well.